src/main/java/worldcities/servlet/WorldCitiesServlet.java

```diff
@@ -1,20 +1,41 @@
 package worldcities.servlet;
 
 import com.google.gson.Gson;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import worldcities.City;
 import worldcities.ParseWorldCities;
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+
 import java.io.IOException;
 
 /**
  * Servlet which given two parameters of long and lat, returns a json object of closest city
  */
 public class WorldCitiesServlet extends HttpServlet {
-
+    ParseWorldCities cities;
     private Gson gson = new Gson();
```

Comment:
You are passing it in or creating it in the constructor, so there's no reason to initialize it here.